windows/shared/logging/PlogTraceLogging/PlogTraceLogging.cxx

```diff
@@ -0,0 +1,13 @@
+
+#include <chrono>
+#include <iomanip>
+#include <sstream>
+
+#include <PlogTraceLogging/PlogTraceLogging.hxx>
```

Comment:
None of these should be needed here, can they be removed? Also, the header that defines `TRACELOGGING_DEFINE_PROVIDER` (`TraceLoggingProvider.h`?) seems to be missing, so that should be added.